apps/web/src/lang/fr.json

```diff
@@ -6646,6 +6646,66 @@
     "defaultMessage": "Mes candidatures",
     "description": "Title for the list of a users applications"
   },
+  "/17wgm": {
+    "defaultMessage": "Rôle(s) ajouté(s) avec succès",
+    "description": "Message displayed to user when one or more roles have been added to a user"
+  },
+  "2lNHxh": {
+    "defaultMessage": "Ajouter un nouveau rôle",
+    "description": "Label for the form to add a role to a user"
+  },
+  "HYYYtr": {
+    "defaultMessage": "L’utilisateur perdra le rôle suivant :",
+    "description": "Text notifying user which role will be removed from the user"
+  },
+  "Hy2UdW": {
+    "defaultMessage": "Vous êtes sur le point de retirer un rôle de l’utilisateur suivant : <strong>{userName}</strong>",
+    "description": "Lead in text for the remove role from user dialog"
+  },
+  "S8ra2P": {
+    "defaultMessage": "Actions",
+    "description": "Title displayed for the role table actions column"
+  },
+  "TvUVa0": {
+    "defaultMessage": "Vous êtes sur le point d’ajouter un nouveau rôle à l’utilisateur suivant : <strong>{userName}</strong>",
+    "description": "Lead in text for the add role to user form."
+  },
+  "UGk7lG": {
+    "defaultMessage": "Enregistrer les changements",
```

Comment:
We have an identical string in English that is translated differently. Which should we use?

https://github.com/GCTC-NTGC/gc-digital-talent/blob/7587d2b1a393b7c19d5f172fec3b7208a7f62d93/apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ApplicationStatusForm/ApplicationStatusForm.tsx#L212-L217

https://github.com/GCTC-NTGC/gc-digital-talent/blob/7587d2b1a393b7c19d5f172fec3b7208a7f62d93/apps/web/src/lang/fr.json#L4641-L4644

Reply:
> Seems like this is a common string we use. Maybe it should be moved to [`packages/i18n/` ](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/packages/i18n/src/messages/formMessages.ts) 🤔

Yes, that makes sense. I'm not sure which one should be used though. @substrae any thoughts?

Reply:
Based on discussion, moved to `packages/i18n` and chose "Enregistrer les modifications" in [extract save changes message into i18n package](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5799/commits/aceaf36882712c223434823c69a6217e814794c6)